static/css/pursuit.css

```diff
@@ -619,6 +619,59 @@ ol li {
 .result__actions__item + .result__actions__item {
   margin-left: 0.65em;
 }
+.result__omission {
```

Comment:
I just need to work out how we will deal with this CSS, because it's generated from a LESS file (in this repo), but there's also a copy of it in the PureScript repo, and they're starting to diverge.